src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs

```diff
@@ -208,7 +208,7 @@ public void ICollection_NonGeneric_CopyTo_TwoDimensionArray_ThrowsException(int
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
+        [ConditionalTheory("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
```

Comment:
One other point, the idea of passing type instead of string to ConditionFact/Theory is good but I started to think to pass the MethodInfo/PropertyInfo instead. what do you think about that?